livekit-agents/livekit/agents/__init__.py

```diff
@@ -16,13 +16,13 @@
 from .job import AutoSubscribe, JobContext, JobProcess, JobRequest
 from .plugin import Plugin
 from .version import __version__
-from .worker import Worker, WorkerOptions
+from .worker import Worker, WorkerOptions, WorkerType
 
 __all__ = [
     "__version__",
     "Worker",
     "WorkerOptions",
-    "WorkerPermissions",
```

Comment:
should we remove?

Reply:
Didn't exist in the imports so figured it wasn't meant to be there

Reply:
dz added it here https://github.com/livekit/agents/pull/625

Reply:
I guess it needs to be imported here https://github.com/livekit/agents/blob/c12589d9cd782ffc28802694ca0b11ffc7c2d0cc/livekit-agents/livekit/agents/worker.py#L83